libs/platform/src/lib/components/thumbnail/thumbnail.component.ts

```diff
@@ -0,0 +1,59 @@
+import { Component, OnInit, Input, ViewEncapsulation, Output, EventEmitter } from '@angular/core';
+
+export interface Media {
+    thumbnailUrl: string;
+    mediaType: string;
+    mediaUrl: string;
+    alt: string;
+    label: string;
+}
+
+export class ThumbnailClickedEvent<T extends ThumbnailComponent = ThumbnailComponent, K = Media> {
+    constructor(
+        /** The source Thumbnail Component of the event. */
+        public source: T,
+        /** The new value of a control. */
+        public payload: K
+    ) { }
+}
+
+@Component({
+    selector: 'fdp-thumbnail',
+    templateUrl: './thumbnail.component.html',
+    styleUrls: ['./thumbnail.component.scss'],
+    encapsulation: ViewEncapsulation.None,
+})
+export class ThumbnailComponent implements OnInit {
+
+    /** variable to input any type of object. */
+    @Input()
+    mediaList: Media[];
+
+    @Input()
+    isHorizontal = false;
+
+    selectedMedia: Media;
```

Comment:
Could you put it under API part? Following [component structure guidelines](https://github.com/SAP/fundamental-ngx/wiki/Code-Contribution-Guideline#component-structure)